src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -67,7 +67,12 @@ def fetch_model_id(model_info_path: str):
     model_info_path = os.path.join(model_info_path, DashboardInfo.MODEL_INFO_FILENAME)
     with open(model_info_path, "r") as json_file:
         model_info = json.load(json_file)
-    return model_info[DashboardInfo.MODEL_ID_KEY]
+    if DashboardInfo.MODEL_ID_KEY not in model_info:
+        raise UserConfigError(
```

Comment:
Can you raise UserConfigValidationException instead of UserConfigError.

Reply:
Also, is it possible to add a unit test for this user error scenario?

Reply:
updated to UserConfigValidationException, there is a issue for unit test with relative import, Kin is trying to see if there are any quick fix, once the issue is fix, I will add unit test. I have created a unit test work item to follow up, thanks 
[Task 2437453](https://msdata.visualstudio.com/Vienna/_workitems/edit/2437453): Add unit test for bug fix <class 'KeyError'> invalid json input in rai_component_utilities.py fetch_model_id()